pkg/minishift/openshift/oc.go

```diff
@@ -17,17 +17,39 @@ limitations under the License.
 package openshift
 
 import (
+	"errors"
 	"fmt"
-	"strings"
 
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	instanceState "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/util"
+	"path/filepath"
+	"regexp"
+	"strings"
 )
 
+type ServiceURL struct {
```

Comment:
I think this does not really belong into _oc.go_. The idea was to provide some generic utility for running any oc command. Maybe we put this into _pkg/minishift/openshift/service/_ package